problems/[230626] 2626. Array Reduce Transformation.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * leetcode problem link: https://leetcode.com/problems/array-reduce-transformation/
+ *
+ * @param {number[]} nums
+ * @param {Function} fn
+ * @param {number} init
+ * @return {number}
+ */
+var reduce = function (nums, fn, init) {
+  if (!nums.length) return init;
+
+  let result;
+  nums.forEach((num) => {
+    result = fn(init, num);
+    init = result;
+  });
+
+  return result;
```

Comment:
```suggestion
  let result = init;
  nums.forEach((num) => {
    result = fn(result, num);
  });

  return result;
```
이렇게 바꾸어서 풀 수도 있습니다~!